jest.config.js

```diff
@@ -4,7 +4,7 @@ module.exports = {
     "^.+\\.vue$": "vue-jest",
     ".+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$":
       "jest-transform-stub",
-    "^.+\\.jsx?$": "babel-jest"
+    "^.+\\.(js|jsx)?$": "babel-jest"
```

Comment:
@jbibla I needed to remove this again as it broke my develop.
This expression also means: Any file with the ending : `.js`, `.jsx` or `.`. I think that is not intended.

Reply:
saw your fix on the other branch. no problem.